examples/README.md

```diff
@@ -2,6 +2,16 @@
 
 These examples demonstrate the main features of Slint and how to use them in different language environments.
 
+### [`carousel`](./carousel)
+
+A custom carousel widget that can be controlled by touch, mouse and keyboard
+
+| `.slint` Design | Rust Source | C++ Source | Node Source | Online wasm Preview | Open in code editor |
+| --- | --- | --- | --- | --- | --- |
+| [`ui.slint`](./carousel/ui/carousel_demo.slint) | [`main.rs`](./carousel/rust/main.rs) | [`main.cpp`](./carousel/cpp/main.cpp) | [`main.js`](./carousel/node/main.js) | [Online simulation](https://slint-ui.com/snapshots/master/demos/carousel/) | [Preview in Online Code Editor](https://slint-ui.com/snapshots/master/editor?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/carousel/ui/carousel_demo.slint) |
```

Comment:
I think we should keep the printerdemo first for now.
Maybe move it after the gallery or the todo?

If we don't have a wasm build online, we can just reference the preview.html like what the iot-dashboard does.